src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -3537,6 +3537,17 @@ void AuraEffect::HandleModArmorPctFromStat(AuraApplication const* aurApp, uint8
     player->UpdateArmor();
 }
 
+void AuraEffect::HandleModStatBonusArmor(AuraApplication const* aurApp, uint8 mode, bool apply) const
+{
+    if (!(mode & (AURA_EFFECT_HANDLE_CHANGE_AMOUNT_MASK | AURA_EFFECT_HANDLE_STAT)))
+        return;
+
+    Unit* target = aurApp->GetTarget();
+
+    if (target->IsPlayer() || target->IsPet())
```

Comment:
im quite sure regular units should also be affected by it since they also have armor. I actually scrolled thru the spells with this aura and kinda only saw things which only players triggers however

Reply:
could be, but spells like 294161 are not casted by pets? or for pets?

Reply:
solved